openssl-sys/src/handwritten/ssl.rs

```diff
@@ -951,3 +951,8 @@ extern "C" {
     #[cfg(any(ossl110, libressl360))]
     pub fn SSL_get_security_level(s: *const SSL) -> c_int;
 }
+
+extern "C" {
+    #[cfg(ossl300)]
+    pub fn SSL_group_to_name(ssl: *const SSL, id: c_int) -> *const c_char;
```

Comment:
This matches the prototype in the documentation, but not the prototype in ssl.h.

Reply:
To be explicit, I think this should fix your compilation issue. Not sure if you need to fix more as a consequence of that:

```suggestion
    pub fn SSL_group_to_name(ssl: *mut SSL, id: c_int) -> *const c_char;
```

Reply:
Addressed with a988bdf1c5376c943a31fe5000174a8ca3fba8e5